src/topology/metric_space/emetric_space.lean

```diff
@@ -754,39 +781,72 @@ end first_countable
 section second_countable
 open topological_space
 
+variables (α)
+
 /-- A separable pseudoemetric space is second countable: one obtains a countable basis by taking
 the balls centered at points in a dense subset, and with rational radii. We do not register
 this as an instance, as there is already an instance going in the other direction
 from second countable spaces to separable spaces, and we want to avoid loops. -/
-lemma second_countable_of_separable (α : Type u) [pseudo_emetric_space α] [separable_space α] :
+lemma second_countable_of_separable [separable_space α] :
   second_countable_topology α :=
 uniform_space.second_countable_of_separable uniformity_has_countable_basis
 
+/-- A sigma compact pseudo emetric space has second countable topology. -/
+lemma second_countable_of_sigma_compact [sigma_compact_space α] :
```

Comment:
If there is a reason not to record this as an instance, can you mention it in the docstring?